src/Examples/Flowsheets/ngcc/soec.py

```diff
@@ -927,95 +819,52 @@ def scale_indexed_constraint(con, sf):
+                self.soec_stack.solid_oxide_cell.oxygen_outlet.mole_frac_comp[t, "CO2"], 1e4
             )
             iscale.set_scaling_factor(
-                self.soec.oxygen_outlet.mole_frac_comp[t, "H2O"], 1e2
+                self.soec_stack.solid_oxide_cell.oxygen_outlet.mole_frac_comp[t, "H2O"], 1e2
             )
             iscale.set_scaling_factor(
-                self.soec.oxygen_outlet.mole_frac_comp[t, "Ar"], 1e2
+                self.soec_stack.solid_oxide_cell.oxygen_outlet.mole_frac_comp[t, "Ar"], 1e2
             )
             iscale.set_scaling_factor(
-                self.soec.oxygen_outlet.mole_frac_comp[t, "N2"], 10
+                self.soec_stack.solid_oxide_cell.oxygen_outlet.mole_frac_comp[t, "N2"], 10
             )
             iscale.set_scaling_factor(
-                self.soec.oxygen_outlet.mole_frac_comp[t, "O2"], 10
+                self.soec_stack.solid_oxide_cell.oxygen_outlet.mole_frac_comp[t, "O2"], 10
             )
 
-        self.soec.recursive_scaling()
-
-        for blk in [
-            self.fuel_inlet_translator,
-            self.fuel_outlet_translator,
-            self.oxygen_inlet_translator,
-            self.oxygen_outlet_translator,
-        ]:
-            shortname = blk.name.split(".")[-1]
-            shortname = "_".join(shortname.split("_")[:2])
-
-            if shortname.endswith("inlet"):
-                props = blk.config.inlet_property_package
-                dest_port = getattr(self.soec, shortname)
-                sflow = props.get_default_scaling("flow_mol")
-                for t in self.time:
-                    iscale.set_scaling_factor(
-                        blk.properties_out[t].flow_mol, sflow * self.number_cells.value
-                    )
-                    iscale.set_scaling_factor(
-                        dest_port.flow_mol[t], sflow * self.number_cells.value
-                    )
-            else:
-                props = blk.config.outlet_property_package
-                sflow = props.get_default_scaling("flow_mol")
-                src_port = getattr(self.soec, shortname)
-                for t in self.time:
-                    iscale.set_scaling_factor(
-                        blk.properties_in[t].flow_mol, sflow * self.number_cells.value
-                    )
-                    iscale.set_scaling_factor(
-                        src_port.flow_mol[t], sflow * self.number_cells.value
-                    )
-
-            scale_indexed_constraint(blk.flow_mol_eqn, sflow)
-
-            sx = props.get_default_scaling("mole_frac_comp")
-            scale_indexed_constraint(blk.mole_frac_eqn, sx)
-
-            sT = props.get_default_scaling("temperature")
-            scale_indexed_constraint(blk.temperature_eqn, sT)
-
-            sP = props.get_default_scaling("pressure")
-            scale_indexed_constraint(blk.pressure_eqn, sP)
+        self.soec_stack.solid_oxide_cell.recursive_scaling()
```

Comment:
The module-level `calculate_scaling_factors` method calls the cell-level `recursive_scaling` after pulling scaling factors from the property packages, so we shouldn't call this method directly when using the module model.